pom.xml

```diff
@@ -21,7 +21,7 @@
 		<rifGeneric.version>4.0.0-SNAPSHOT</rifGeneric.version>
 		<junit.version>4.11</junit.version>
 		<sqlserver.version>6.4.0.jre8</sqlserver.version>
-		<postgres.version>9.3-1102-jdbc3</postgres.version>
+		<postgres.version>42.2.5</postgres.version>
```

Comment:
This version number seems crazily high for Postgres. Is it actually the driver version, since the old one includes `jdbc3` on the end?

Reply:
As of Postgres 9.5 the JDBC driver number changed to 42.0.0!